packages/core/backend/sirius-components-collaborative/src/main/java/org/eclipse/sirius/components/collaborative/dto/ToolVariable.java

```diff
@@ -10,7 +10,7 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-package org.eclipse.sirius.components.collaborative.diagrams.dto;
+package org.eclipse.sirius.components.collaborative.dto;
```

Comment:
I'm failing to see the GraphQL type `ToolVariable` moved from diagram to core parts of the GraphQL schema.